contributing.md

```diff
@@ -1,70 +1,80 @@
 # Contributing to the Playbook
 
 ## What you will need
+
 1. A GitHub account
-2. To be invited to the [dxw organisation](https://github.com/dxw)
-3. To be invited to the [staff team](https://github.com/orgs/dxw/teams/staff)
+1. To be invited to the [dxw organisation](https://github.com/dxw)
```

Comment:
This seems weird – I know it's valid Markdown, but it stops it scanning nicely in plain text format. Is it down to a particular rule we might be able to disable? What do others think?

Reply:
The argument for doing it as all `1.` is that adding and removing and reordering items is less painful. the rendered version will do the right thing.
I personally think that actually all ordered lists should just be unordered ones since do the numbers actually add anything.

Reply:
Seems reasonable to me. Definitely agree on using unordered lists in most places.